### PR TITLE
Remove hardcoded AWS bucket Name from a Few Files

### PR DESCRIPTION
**Pull request by mstruve**
## What type of PR is this? (check all applicable)
- [x] Refactor

## Description
This is only a small number of the places we have hardcoded our old AWS bucket name. I figured I would do these and then make an Issue to handle all of the ones that are in the view and emails which are A LOT and will need some closer attention as some are cloudinary links and some point straight to S3. 

## Related Ticket
Followup for #5771 and #5555 
 
## Added to documentation?
- [x] no documentation needed

alt_text

### CODE REVIEW

File: app/labor/profile_image.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class ProfileImage
   def initialize(resource)
     @resource = resource
     @image_link = resource.profile_image_url
-    @backup_link = "https://thepracticaldev.s3.amazonaws.com/i/99mvlsfu5tfj9m7ku25d.png"
+    @backup_link = "https://#{ApplicationConfig['AWS_BUCKET_NAME']}.s3.amazonaws.com/i/99mvlsfu5tfj9m7ku25d.png"
```
Comment (rhymes) [inline app/labor/profile_image.rb]: wouldn't this break local installations when merged? 
For example:

- the local `AWS_BUCKET_NAME` is not `thepracticaldev` by default, it's `Optional`
- mine is called `devto-tests` because I configured it to point to my AWS installation
- this would mean that I'd have to manually upload a file named `99mvlsfu5tfj9m7ku25d.png` for this to work

I think it'd have an impact on the other communities as well.

Maybe we could keep this hardcoded until we find a better way to handle this:

- either we download these images and put them in our assets (my fav option)
- or we find a way to upload them when the app is setup, through a script or something

Basically I don't think these images should be in AWS at all :)

File: app/lib/html_css_to_image.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ module HtmlCssToImage
   AUTH = { username: ApplicationConfig["HCTI_API_USER_ID"],
            password: ApplicationConfig["HCTI_API_KEY"] }.freeze
 
-  FALLBACK_IMAGE = "https://thepracticaldev.s3.amazonaws.com/i/g355ol6qsrg0j2mhngz9.png".freeze
+  FALLBACK_IMAGE = "https://#{ApplicationConfig['AWS_BUCKET_NAME']}.s3.amazonaws.com/i/g355ol6qsrg0j2mhngz9.png".freeze
```
Comment (rhymes) [inline app/lib/html_css_to_image.rb]: same as the previous observation

File: app/models/site_config.rb [hunk 3]
```diff
@@ -14,7 +14,7 @@ class SiteConfig < RailsSettings::Base
   field :social_networks_handle, type: :string, default: "thepracticaldev"
 
   # images
-  field :main_social_image, type: :string, default: "https://thepracticaldev.s3.amazonaws.com/i/6hqmcjaxbgbon8ydw93z.png"
+  field :main_social_image, type: :string, default: "https://#{ApplicationConfig['AWS_BUCKET_NAME']}.s3.amazonaws.com/i/6hqmcjaxbgbon8ydw93z.png"
```
Comment (rhymes) [inline app/models/site_config.rb]: he're we're definitely breaking the main social image for a pristing installation

File: app/services/moderator/banish_user.rb [hunk 4]
```diff
@@ -46,7 +46,7 @@ def remove_profile_info
         twitch_url: nil, feed_url: nil
       )
 
-      user.update_columns(profile_image: "https://thepracticaldev.s3.amazonaws.com/i/99mvlsfu5tfj9m7ku25d.png")
+      user.update_columns(profile_image: "https://#{ApplicationConfig['AWS_BUCKET_NAME']}.s3.amazonaws.com/i/99mvlsfu5tfj9m7ku25d.png")
```
Comment (rhymes) [inline app/services/moderator/banish_user.rb]: this is further evidence we might want to rethink this approach, we are storing a broken path in the DB:

- it's either a broken URL for new installers
- or a URL that points to non existent images if the bucket name is not `thepracticaldev`

**Comment by mstruve** [comment]
@rhymes you are right, a lot of these images should be downloaded and put in the assets folder. Since there is a bit more scope to this than I anticipated I am going to close and make an issue to get it sorted out.

**Comment by mstruve** [comment]
@rhymes I wasn't sure what labels to put on it but here is the new issue https://github.com/thepracticaldev/dev.to/issues/5810

**Comment by rhymes** [comment]
@mstruve  went ahead and put them all haahaahha
